crates/core/c8y_api/src/smartrest/message_ids.rs

```diff
@@ -0,0 +1,107 @@
+//! Definitions of Smartrest MQTT message ids.
```

Comment:
Module name suggestion: `static_templates.rs`(https://cumulocity.com/docs/smartrest/mqtt-static-templates)

Reply:
It was called that previously, but also wanted to incorporate ids like `41` and `71`, which were used in the codebase but aren't really templates.